setup.py

```diff
@@ -101,6 +101,12 @@ def __init__(self, dist):
 
 
 class BuildExtraLibraries(BuildExtCommand):
+    def finalize_options(self):
+        self.distribution.ext_modules[:] = [
```

Comment:
Assigning to `[:]` will expand the input, so you don't actually need the `[*]` around `filter`.

Reply:
good point, changed